### PR TITLE
HookTest 1.0.0

### PR DESCRIPTION
**Pull request by PonteIneptique**
It will need to be squashed though :)

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +58236 |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/152)*

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ python:
   - '3.5'
 
 install:
-  - pip3 install HookTest==0.2.2
+  - pip3 install HookTest>=1.0.0
 
-script: hooktest --scheme epidoc --workers 3 --verbose --travis --countword ./ --from_travis_to_hook "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/First1KGreek"
+script: hooktest ./ --console table --scheme epidoc --workers 3 --verbose 10 --manifest --countword --hookUI "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/First1KGreek" --allowfailure
```
Comment (sonofmun) [inline .travis.yml]: You should take --allowfailure out of the command since this is a 100% repo.

**Comment by PonteIneptique** [comment]
Done

**Comment by capitains-hook** [comment]
## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +58236 |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/221)*

**Comment by sonofmun** [comment]
Same question with verbosity here as with csel. Maybe set it to 7 instead?

**Comment by capitains-hook** [comment]
## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +58236 |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/225)*

**Pull request merged**